commit 380fab4bc1507d8a4422ad69192c045e06853c88
Author: Nico Isenbeck <nico.isenbeck@web.de>
Date:   Wed Mar 30 19:13:33 2016 +0000

    ARM: dts: sun7i: Enable audio codec on BananaPro
    Enable the on-chip audio codec

diff --git a/arch/arm/boot/dts/sun7i-a20-bananapro.dts b/arch/arm/boot/dts/sun7i-a20-bananapro.dts
index 18fcc87..f1a3356 100644
--- a/arch/arm/boot/dts/sun7i-a20-bananapro.dts
+++ b/arch/arm/boot/dts/sun7i-a20-bananapro.dts
@@ -104,6 +104,10 @@
        status = "okay";
 };
 
+&codec {
+       status = "okay";
+};
+
 &ehci0 {
        status = "okay";
 };

